packages/preview/zen-zine/0.2.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Tom Eichlersmith
```

Comment:
```suggestion
Copyright (c) 2024-2025 Tom Eichlersmith
```

Reply:
Thank you for catching this! I totally forgot :)